doc/source/cornwall_gov_uk.md

```diff
@@ -17,13 +17,13 @@ waste_collection_schedule:
 
 ### Configuration Variables
 
-**postcode**  
+**postcode**
 *(string) (optional)*
 
-**hournameornumber**  
+**housenameornumber**
```

Comment:
Not the same as.
https://github.com/mampfes/hacs_waste_collection_schedule/blob/1f53eaf87b22de8778deaa202ae3a3455eefac41/custom_components/waste_collection_schedule/waste_collection_schedule/source/cornwall_gov_uk.py#L22-L25
 Would be nice if you could rename this to  housenumberorname (here and in yaml parts). changeing the python might break existing configurations

Reply:
Done